static/src/javascripts/projects/commercial/modules/ipsos-mori.js

```diff
@@ -25,15 +27,23 @@ const loadIpsosScript = () => {
 };
 
 export const init = () => {
-	getLocale().then((locale) => {
-		if (locale === 'GB') {
-			onConsentChange((state) => {
-				if (getConsentFor('ipsos', state)) {
-					return loadIpsosScript();
-				}
-			});
-		}
-	});
-
+	getLocale()
```

Comment:
This promise chain here might be iffy?

`init` returns a `Promise.resolve()` but internally is doing some async script loading.

Should we be chaining the real promise or do we not care about the async internals.

Reply:
I think this can/will happen once we move this to TypeScript?

Reply:
To make the PRs a bit simpler, my plan was to avoid modifying the return values of each `init` function in this PR.
Then in a subsequent PR we'll change all of the functions that return `Promise.resolve()` so that they return a promise which represents the functions actually completing.